src/applications/personalization/dashboard/containers/ClaimsAppealsWidget.jsx

```diff
@@ -51,8 +50,6 @@ class ClaimsAppealsWidget extends React.Component {
     if (this.props.canAccessAppeals) {
       this.props.getAppealsV2();
     }
-
-    scrollToTop();
```

Comment:
This was redundant. This component is only mounted by `DashboardApp` which also scrolls to top when it loads.